src/main/java/be/syntra/devshop/DevshopBack/services/UserServiceImpl.java

```diff
@@ -117,4 +119,11 @@ public Authentication changeUserPassword(User currentUser, String newPassword) {
         SecurityContextHolder.getContext().setAuthentication(authenticationManagerBuilder.getObject().authenticate(authenticationToken));
         return authenticationToken;
     }
+
+    public User getUserById(Long userId) {
+        return userRepository.findById(userId)
+                .stream()
```

Comment:
unnecessary stream(). Change to userRepository.findById(userId).orElseThrow(() -> UserNotFoundException(""));

Reply:
fixed